api/nakadi-event-bus-api.yaml

```diff
@@ -606,22 +488,389 @@ paths:
+          required: true
+      responses:
+        '200':
+          description: Offsets were committed
+        '204':
+          description: |
+            No Content. At least one cursor which was tried to be committed is older or equal to already committed one.
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Client is not authorized to perform this operation
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Subscription not found
+          schema:
+            $ref: '#/definitions/Problem'
+        '422':
+          description: Unprocessable Entity
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Service (temporarily) unavailable
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /subscriptions/{subscription_id}/events:
+    get:
+      tags:
+        - stream-api
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Starts a new stream for reading events from this subscription. The data will be automatically rebalanced
+        between streams of one subscription. The minimal consumption unit is a partition, so it is possible to start as
+        many streams as the total number of partitions in event-types of this subscription. The rebalance currently
+        only operates with the number of partitions so the amount of data in event-types/partitions is not considered
+        during autorebalance.
+        The position of the consumption is managed by Nakadi. The client is required to commit the cursors he gets in
+        a stream. This also depends on the `commit_mode` parameter that is set for the stream.
+      parameters:
+        - $ref: '#/parameters/SubscriptionId'
+        - name: commit_timeout
+          in: query
+          description: |
+            The amount of time allowed between commits. If no commit is performed within this amount of time
+            - the connection will be closed from Nakadi side. Specified in seconds.
+          type: integer
+          format: int32
+          required: false
+          default: 30
+        - name: window_size
+          in: query
+          description: |
+            The amount of uncommitted events Nakadi will stream before pausing the stream. When in paused
+            state and commit comes - the stream will continue. Minimal window size is 1.
+          type: integer
+          format: int32
+          required: false
+          default: 10000
+        - name: commit_mode
+          in: query
+          description: |
+            The commit mode that will be used for this stream. Currently only `commit_at_will` is supported.
```

Comment:
@whiskeysierra No, we don't have ETA for this yet.
We have WIP implementation and there we have only 'commit_at_will' implemented. And for the first version we planned to have only it.
Are you interested in using other modes?